zombienet_tests/functional/0001-basic-network.zndsl

```diff
@@ -0,0 +1,15 @@
+Description: Small Network test
```

Comment:
why the `functional` folder?

Reply:
I was replicating what was done on the polkadot repo.